pkg/controller/controller.go

```diff
@@ -155,7 +159,7 @@ func (c *controller) Run(workers int, stopCh <-chan struct{}) {
 		go wait.Until(worker(c.brokerQueue, "ServiceBroker", maxRetries, true, c.reconcileServiceBrokerKey), time.Second, stopCh)
 		go wait.Until(worker(c.serviceClassQueue, "ServiceClass", maxRetries, true, c.reconcileServiceClassKey), time.Second, stopCh)
 		go wait.Until(worker(c.instanceQueue, "ServiceInstance", maxRetries, true, c.reconcileServiceInstanceKey), time.Second, stopCh)
-		go wait.Until(worker(c.bindingQueue, "ServiceInstanceCredential", maxRetries, true, c.reconcileServiceInstanceCredentialKey), time.Second, stopCh)
+		go wait.Until(worker(c.bindingQueue, "ServiceInstanceCredential", math.MaxInt32, true, c.reconcileServiceInstanceCredentialKey), time.Second, stopCh)
```

Comment:
Just curious as to why credentials have no retry count limit, but other resources do.

Reply:
Oversight as part of my initial work. I meant to change them all (or really remove the variable completely).